modules/social_features/social_content_report/social_content_report.install

```diff
@@ -81,6 +81,7 @@ function social_content_report_install() {
       'create terms in report_reasons',
       'delete terms in report_reasons',
       'edit terms in report_reasons',
+      'administer flaggings',
```

Comment:
@Macsym2018 Could you explain why permission `view inappropriate reports` is not enough and we also need to grant `administer flaggings` ?

Reply:
@ribel 
We have the function sendMassImmediately and there we have access check to view this entity
if ($parameters['activity']->getRelatedEntity()->access('view', $target_account)) {
Access checking in the access function in ContentEntityBase
![image](https://github.com/user-attachments/assets/13180e66-17dd-4810-9b0d-1184b7fa10cd)
And again move us to EntityAccessControlHandler
![image](https://github.com/user-attachments/assets/50763692-0418-4bf4-b4f1-3574f150244e)
And there we check access in the checkAccess
![image](https://github.com/user-attachments/assets/0935ef9f-a84a-4cc7-b138-af19d1f21fdd)
There we have the condition if ($admin_permission = $this->entityType->getAdminPermission()) {
There we check if a user has Admin permissions
![image](https://github.com/user-attachments/assets/c74083c2-635a-4279-8226-8c08c93e29b3)
It means if we want to receive emails. The "administer flagging" permission should be granted to the SM role.